webplugin/js/app/zendesk-chat-service.js

```diff
@@ -5,9 +5,11 @@ function ZendeskChatService() {
     var _this = this;
     var ZENDESK_SDK_INITIALIZED = false;
     var ZENDESK_CHAT_SDK_KEY = "";
+    var preChatLeadData = "";
```

Comment:
when assigning something to it, are we assigning Object or String?
we need to keep base value same as the final value in terms of data type